components/EntireScheme.svelte

```diff
@@ -81,7 +81,7 @@
     // TODO Should we prompt before deleting the current scheme?
     reader.onload = (e) => {
       try {
-        gjScheme.set(backfill(JSON.parse(e.target.result)));
+        gjScheme.set(backfill(JSON.parse(e.target.result as string)));
```

Comment:
It could also be `ArrayBuffer`; the type checker's not smart enough to handle the `readAsText` below, so a type assertion is apparently the normal response